### PR TITLE
Triforce

### PR DESCRIPTION
**Pull request by chris0**
**What does your Instant Answer do?**
Shows key combinations for creating the Tri Force symbol on Lunix, Mac, Windows

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.ehow.com/how_8460794_triforce.html

**Why did you choose this data source?**
Requested: https://duck.co/ideas/idea/6134/how-to-tri-force-instructional

**Are there any other alternative (better) data sources?**
None that I know of

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers, programmers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?** Yes, https://duck.co/ideas/idea/6134/how-to-tri-force-instructional

**Are you having any problems? Do you need our help with anything?**
Could not test on duckpan server

**Where did you hear about DuckDuckHack? (For first time contributors)**
technical.ly/phily

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Was unable to test but validated json

IA Page: https://duck.co/ia/view/triforce_cheat_sheet

### CODE REVIEW

**Comment by mintsoft** [comment]
Hiya @chris0 can you copy and paste the cheatsheet PR template into the description (https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and fill it out?

**Comment by chris0** [comment]
## Copy and Paste the Raw Text Below Into Your Pull Request

**What does your Instant Answer do?**
Shows key combinations for creating the Tri Force symbol on Lunix, Mac, Windows


**What is the data source for your Instant Answer? (Provide a link if possible)**
http://www.ehow.com/how_8460794_triforce.html

**Why did you choose this data source?**
Requested: https://duck.co/ideas/idea/6134/how-to-tri-force-instructional

**Are there any other alternative (better) data sources?**
None that I know of

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers, programmers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?** Yes, https://duck.co/ideas/idea/6134/how-to-tri-force-instructional


**Are you having any problems? Do you need our help with anything?**
Could not test on duckpan server

**Where did you hear about DuckDuckHack? (For first time contributors)**
technical.ly/phily

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Was unable to test but validated json

**Comment by chris0** [comment]
Hi @mintsoft, I was unable to test on the duckpan server but I validated the json.

File: share/goodie/cheat_sheets/json/triforce.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+	"id": "triforce_cheat_sheet",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/triforce.json]: Can you switch from hard-tabs to 4 spaces?

**Comment by mintsoft** [comment]
@chris0 it works for me when I run `duckpan server`; I think you might be better flipping the values of `key` and `value` as the template looks a little funny as is:

image

**Comment by chris0** [comment]
@mintsoft glad to see it's showing up. I'll flip those key:values

**Comment by chris0** [comment]
changed tabs to spaces

**Comment by mintsoft** [comment]
Hmm it looks better, but I'm still not too sure what we can do to make it look less crazy.

@moollaza @abeyang any thoughts?
image

**Comment by chris0** [comment]
Are the column titles coming from "section_order": []? Should I leave the vals empty?

**Comment by mintsoft** [comment]
@chris0 that's where the bold ones come from yes.

What's the reason you're unable to run `duckpan server` btw?

**Comment by chris0** [comment]
@mintsoft I can run the server but I just didn't see the instant answer appear after typing in a query string such as 'cheat sheet triforce'

Maybe if I leave the vals as empty string and break the keystrokes into three steps, that will clean it up a little.

**Comment by chris0** [comment]
@mintsoft I didn't create a trigger, could that be the problem?

**Comment by mintsoft** [comment]
@chris0 try `triforce cheatsheet` instead

**Comment by mintsoft** [comment]
@chris0  The triggers should be automatic for cheatsheets, we have one IA that implements all the cheatsheets

**Comment by chris0** [comment]
@mintsoft  Eureka! It does work!

**Comment by mintsoft** [comment]
Excellent :) I think we changed the triggers a few months back so they only work with `cheatsheet` at the end rather than the start

**Comment by chris0** [comment]
A couple alternatives:
1. leaving vals blank
2. breaking up into additional steps

formatting

**Comment by mintsoft** [comment]
Hmm, none of them seem to work excellently. @abeyang any ideas?

**Comment by abeyang** [comment]
@mintsoft @chris0 I don't have any brilliant ideas for this, either. Of the different variations, I prefer the breaking up into additional steps (though I think we should be explicit about the steps, eg, "Step 1").

File: share/goodie/cheat_sheets/json/triforce.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	"name": "How To Tri-Force Instructional",
+	"description": "Step-by-step Tri-Forcing guide for Windows, Mac and Linux computers.",
+	"metadata": {
+		"sourceName": "eHow.com",
+		"sourceUrl": "http://www.ehow.com/how_8460794_triforce.html"
+	},
+	"aliases": ["triforce", "tri-force", "tri force"],
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/triforce.json]: You can remove "triforce" alias, as that already comes from the filename

**Comment by moollaza** [comment]
Continued in #1550

**Pull request closed without merging**